plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -259,7 +259,7 @@ export class EventPipelineRunner {
             event.team_id
         )
 
-        if (event.event === '$exception' && event.team_id == 2) {
+        if (event.event === '$exception' && !event.properties?.hasOwnProperty('$sentry_event_id')) {
```

Comment:
I think this also works but it might be more idiomatic to do something like:
```suggestion
        if (event.event === '$exception' && !('$sentry_event_id' in event.properties)) {
```

Reply:
I'd have to do something like `!('$sentry_event_id' in (event.properties ?? {}))`, which doesn't really feel better, but I don't have any sense for what's tasteful in typescript really